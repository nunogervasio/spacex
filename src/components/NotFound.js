import React from "react";
import monster from "../img/alien-with-aqualung.svg";
const NotFound = () => {
  return (
    <div className="not-found">
      <h1>Page Cannot Be Found</h1>
      <p className="space-font">Only...</p>
      <img src={monster} className="monster" />
      <h2>Space Monster!</h2>
    </div>
  );
};

export default NotFound;
