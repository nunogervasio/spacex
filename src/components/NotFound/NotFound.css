.monster {
  max-width: 15em;
}
.not-found {
  display: flex;
  flex-direction: column;
  align-items: center;
}
.space-font {
  text-align: center;
  font-family: "Staatliches", cursive;
}
