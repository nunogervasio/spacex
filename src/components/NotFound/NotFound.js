import React from "react";
import monster from "../../img/alien-with-aqualung.svg";
import "./NotFound.css";

const NotFound = () => {
  return (
    <div className="not-found">
      <h1>Page Cannot Be Found</h1>
      <p className="space-font">Only...</p>
      <img src={monster} alt="space monster" className="monster" />
      <h2>Space Monster!</h2>
    </div>
  );
};

export default NotFound;
