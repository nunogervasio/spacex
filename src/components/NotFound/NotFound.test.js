import React from "react";
import { shallow } from "enzyme";
import NotFound from "./NotFound";

describe("Footer", () => {
  const notFound = shallow(<NotFound />);
  it("renders properly", () => {
    expect(notFound).toMatchSnapshot();
  });
});
