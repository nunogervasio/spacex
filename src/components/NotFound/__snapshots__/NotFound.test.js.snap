// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Page Cannot Be Found
        </h1>,
        <p
          className="space-font"
        >
          Only...
        </p>,
        <img
          alt="space monster"
          className="monster"
          src="alien-with-aqualung.svg"
        />,
        <h2>
          Space Monster!
        </h2>,
      ],
      "className": "not-found",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Page Cannot Be Found",
        },
        "ref": null,
        "rendered": "Page Cannot Be Found",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Only...",
          "className": "space-font",
        },
        "ref": null,
        "rendered": "Only...",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "space monster",
          "className": "monster",
          "src": "alien-with-aqualung.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Space Monster!",
        },
        "ref": null,
        "rendered": "Space Monster!",
        "type": "h2",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Page Cannot Be Found
          </h1>,
          <p
            className="space-font"
          >
            Only...
          </p>,
          <img
            alt="space monster"
            className="monster"
            src="alien-with-aqualung.svg"
          />,
          <h2>
            Space Monster!
          </h2>,
        ],
        "className": "not-found",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Page Cannot Be Found",
          },
          "ref": null,
          "rendered": "Page Cannot Be Found",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Only...",
            "className": "space-font",
          },
          "ref": null,
          "rendered": "Only...",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "space monster",
            "className": "monster",
            "src": "alien-with-aqualung.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Space Monster!",
          },
          "ref": null,
          "rendered": "Space Monster!",
          "type": "h2",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
